.github/actions/ppml-spark-local-example-tests-on-gramine-action/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
         DEFAULT_SGX_MEM_SIZE: 32G
         DEFAULT_IMAGE: ${{ inputs.image }}:${{ inputs.image-tag }}
       run: |
-        echo "CONTAINER_NAME=spark-exmaples-test-gramine" >> $GITHUB_ENV
+        echo "CONTAINER_NAME=spark-examples-test-gramine" >> $GITHUB_ENV
```

Comment:
Could you put this action into .github/actions/ppml/ folder? 
`.github/actions/ppml/ppml-spark-local-example-tests-on-gramine-action/action.yml`